Frontend/android-native/mobility-common/src/main/java/in/juspay/mobility/common/MobilityCommonBridge.java

```diff
@@ -1656,6 +1677,69 @@ protected Bitmap getMarkerBitmapFromView(String locationName, String imageName,
         return returnedBitmap;
     }
 
+    protected void onMapUpdate() {
+        try {
+            MapRemoteConfig mapRemoteConfig = getMapRemoteConfig();
+            if(mapRemoteConfig.enableMapRecenter) {
+                String merchant = bridgeComponents.getContext().getResources().getString(in.juspay.mobility.app.R.string.service);
+                String merchantType = merchant.contains("partner") || merchant.contains("driver") || merchant.contains("provider") ? "DRIVER" : "USER";
+                if (merchantType.equals("USER")) {
```

Comment:
Why it is not added for driver? create a ENUM and variable in Common and assign the value in `MobilityCustomerBridge` and `MobilityDriverBridge` constructor as CONSUMER or PROVIDER
.